website/docs/guides/duckdb-qs.md

```diff
@@ -0,0 +1,309 @@
+icon: 'duckdb-seeklogo'
+level: 'Beginner'
+hide_table_of_contents: true
+tags: ['dbt Core','Quickstart']
+---
+
+<div style={{maxWidth: '900px'}}>
+
+## Introduction
+
+In this quickstart guide, you'll learn how to use dbt Core with DuckDB, enabling you to get set up quickly and efficiently. [DuckDB](https://duckdb.org/) is an open-source database management system which is designed for analytical workloads. It is designed to provide fast and easy access to large datasets, making it well-suited for data analytics tasks. 
+
+
+This guide will demonstrate how to: 
+
+- [Create a virtual development environment](/docs/core/pip-install#using-virtual-environments) using a template provided by dbt Labs.
+   - This sets up a fully functional dbt environment with an operational and executable project. The codespace automatically connects to the DuckDB database and loads a year's worth of data from our fictional Jaffle Shop café, which sells food and beverages in several US cities.
+   - For additional information, refer to the [README](https://github.com/dbt-labs/jaffle_shop_duckdb/blob/duckdb/README.md) for the Jaffle Shop template. It includes instructions on how to do this, along with animated GIFs.
+- Run any dbt command from the environment’s terminal. 
+- Generate a larger dataset for the Jaffle Shop café (for example, five years of data instead of just one).
+
+You can learn more through high-quality [dbt Learn courses and workshops](https://learn.getdbt.com). 
+
+
+### Related content
+
+
+- [DuckDB setup](/docs/core/connect-data-platform/duckdb-setup)
+- [Create a GitHub repository](/guides/manual-install?step=2)
+- [Build your first models](/guides/manual-install?step=3)
+- [Test and document your project](/guides/manual-install?step=4)
+
+
+## Prerequisites
+
+- When using DuckDB with dbt Core, you'll need to use the dbt command-line interface (CLI). Currently, DuckDB is not supported in dbt Cloud.
+- It's important that you know some basics of the terminal. In particular, you should understand `cd`, `ls` , and `pwd` to navigate through the directory structure of your computer easily.
+- You have a [GitHub account](https://github.com/join).
+
+## Set up DuckDB for dbt Core
+
+This section will provide a step-by-step guide for setting up DuckDB for use in local (Mac and Windows) environments and web browsers.
+
+In the repository, there's a [`requirements.txt`](https://github.com/dbt-labs/jaffle_shop_duckdb/blob/duckdb/requirements.txt) file which is used to install dbt Core, DuckDB, and all other necessary dependencies. You can check this file to see what will be installed on your machine. It's typically located in the root directory of your project.
+
+The `requirements.txt` file is placed at the top level of your dbt project directory, alongside other key files like `dbt_project.yml`:
+
+
+```shell
+
+/my_dbt_project/
+├── dbt_project.yml
+├── models/
+│   ├── my_model.sql
+├── tests/
+│   ├── my_test.sql
+└── requirements.txt
+
+```
```

Comment:
This whole section is kind of confusing. It is worded as if you need to take this file and put it in your repository, but that isn't true. It is in the repository already, and you just have to invoke it to bring in any necessary dependencies. I think this should be more of a callout, like, "hey we are going to install some dependencies below, if you want to check those out first, go to this file in the repo, otherwise, we will show you how in later steps."